src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -163,6 +163,34 @@ public void MulticastInterface_Set_InvalidIndex_Throws()
             await MulticastInterface_Set_IPv6_Helper(0);
         }
 
+        [Fact]
+        public void MulticastTTL_Set_IPv4_Succeeds()
+        {
+            using ( Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp))
```

Comment:
nit: remove extra space before "Socket socket".  And actually, this can be "var socket" because the type is obvious since it is doing a constructor.